### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by scott-wilson** [bh/atom-python-isort#3]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (c:\Users\scott.wilson\AppData\Local\atom\app-0.174.0\resources\app\src\atom.js:55:11)
Object.activate (c:\Users\scott.wilson\.atom\packages\python-isort\lib\index.coffee:12:9)
```

**Comment by MateuszG** [comment]
+1

**Comment by jmfederico** [comment]
Fixed in commit e6c83067c29a21815fd3bfbe596fac84ff731294
